doc/man3/EVP_PKEY_fromdata.pod

```diff
@@ -50,9 +61,93 @@ EVP_PKEY_fromdata() return 1 for success and 0 or a negative value for
 failure.  In particular a return value of -2 indicates the operation is
 not supported by the public key algorithm.
 
+=head1 EXAMPLES
+
+=head2 Creating an RSA keypair
```

Comment:
I think an example for a second, different key type might be useful, so the reader can see what is the same and what is different between different types of key.

Reply:
Do you mean a key type that we know about, or the fictitious BLARGH?

Reply:
I'd suggest a real key type. Say X25519 or ECDH or...whatever.

Reply:
ECDH isn't a key type...

Reply:
Well ok, EC then.

Reply:
Added an ECC example